### PR TITLE
Delete program api test

### PR DESCRIPTION
**Pull request by RubenGeo**
[AB#32012](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/32012) 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: services/121-service/test/program/delete-program.test.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  programIdPV,
+  registrationCbe,
+  registrationsOCW,
+  registrationsPV,
+} from '@121-service/test/registrations/pagination/pagination-data';
+
+describe('Delete program', () => {
+  let accessToken: string;
+
+  it('should delete a nlrc programs', async () => {
```
Comment (jannisvisser) [inline services/121-service/test/program/delete-program.test.ts:25]:
```suggestion
  it('should delete nlrc programs', async () => {
```

File: services/121-service/test/program/delete-program.test.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    expect(deleteResponsePV.statusCode).toBe(HttpStatus.NO_CONTENT);
+
+    const getProgramResponseOCW = await getProgram(programIdOCW, accessToken);
+    expect(getProgramResponseOCW.statusCode).toBe(HttpStatus.NOT_FOUND);
+
+    const getProgramResponsePV = await getProgram(programIdPV, accessToken);
+    expect(getProgramResponsePV.statusCode).toBe(HttpStatus.NOT_FOUND);
+  });
+
+  it('should delete eth programs', async () => {
```
Comment (jannisvisser) [inline services/121-service/test/program/delete-program.test.ts:58]: why ETH, and not also others (KRCS with safaricom-transfer.entity). (I saw your comment elsewhere about the eslint rule already catching a lot)

File: services/121-service/test/program/delete-program.test.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  registrationsOCW,
+  registrationsPV,
+} from '@121-service/test/registrations/pagination/pagination-data';
+
+describe('Delete program', () => {
+  let accessToken: string;
+
+  it('should delete a nlrc programs', async () => {
+    // Arrange
+    await resetDB(SeedScript.nlrcMultiple);
```
Comment (jannisvisser) [inline services/121-service/test/program/delete-program.test.ts:27]: why not use nlrcMultipleSeecMock? I guess it doesn't matter much..

File: services/121-service/test/program/delete-program.test.ts (hunk 2 above)
Comment (RubenGeo) [reply services/121-service/test/program/delete-program.test.ts:58]: I started with ETH and than I started to have doubts about the usefulness of doing it for all Fsps/ programs. But ETH was already done, so I left it..

File: services/121-service/test/program/delete-program.test.ts (hunk 3 above)
Comment (RubenGeo) [reply services/121-service/test/program/delete-program.test.ts:27]: Yes, I thought this was easier to read in an API tests. You need less context to understand what happens, but I don't really have a strong opinion about it

Comment (jannisvisser) [reply services/121-service/test/program/delete-program.test.ts:27]: fine

File: services/121-service/test/program/delete-program.test.ts (hunk 2 above)
Comment (jannisvisser) [reply services/121-service/test/program/delete-program.test.ts:58]: Ok I guess it's fine indeed. With the eslint present.

**Review by jannisvisser — APPROVED** [review]

**Pull request merged**